benefits/static/css/admin/styles.css

```diff
@@ -58,3 +58,16 @@ html[data-theme="light"],
   --link-fg: #ffffff;
   --body-quiet-color: var(--dark-color);
 }
+
+#user-tools,
+#logout-form button {
+  padding: 0;
+  font-weight: 300;
+  font-size: 0.6875rem;
+  letter-spacing: 0.5px;
```

Comment:
I believe this CSS Is unneccessary. These styles are already being set by `base.css`. The only one necessary is `text-transform: unset`.

Reply:
If we're okay with `Sign Out` instead of `Sign out`, we can use Bootstrap's `text-capitalize` class instead. Unfortunately there is no `text-unset` class for `text-transform: unset`...... Then we could get all the styles in via utility classes (the preferred method).

Reply:
That's true, thanks for pointing that out. I only left the necessary CSS so that we match the capitalization in the mock-up (`Sign out`).